Sources/Contour/Configurator/AppConfigConfigurator.cs

```diff
@@ -230,36 +235,83 @@ public IBusConfigurator Configure(string endpointName, IBusConfigurator cfg)
                     senderCfg.WithRequestTimeout(message.Timeout);
                 }
             }
+            #endregion
+
+            #region Incoming
 
-            foreach (IncomingElement message in endpointConfig.Incoming)
+            foreach (IncomingElement incomingElement in endpointConfig.Incoming)
             {
-                IReceiverConfigurator receiver = cfg.On(message.Label).
-                    WithAlias(message.Key);
+                var configurator = cfg.On(incomingElement.Label).
+                    WithAlias(incomingElement.Key);
+
+                //this should be the default value
+                var qos = configurator.GetQoS();
+
+                if (qos.HasValue)
+                {
+                    var size = qos.Value.PrefetchSize;
+                    var count = qos.Value.PrefetchCount;
+                    
+                    if (endpointConfig.Qos.PrefetchSize.HasValue)
```

Comment:
Yes and this should be changed. But it should only be available for RabbitMQ transport configuration.